package.json

```
@@ -9,7 +9,7 @@
     "lint": "eslint",
     "lint:fix": "eslint --fix",
     "test": "npm run test:unit && npm run test:typescript",
-    "test:unit": "tap",
+    "test:unit": "c8 node --test",
```

One last thing, was the coverage not 100%?

If so, please add this:

```suggestion
    "test:unit": "c8 --100 node --test",
```

---

No It wasn't, that's why I haven't configured it